tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -332,6 +332,6 @@ def objective(trial: Trial) -> float:
             datetime_start = trials[i + 1].datetime_start
             prev_datetime_complete = trials[i].datetime_complete
             assert datetime_start is not None and prev_datetime_complete is not None
-            trial_prep = (datetime_start - prev_datetime_complete).seconds
-            heartbeats_interval = (trial_heartbeats[i + 1] - trial_heartbeats[i]).seconds
+            trial_prep = (datetime_start - prev_datetime_complete).total_seconds()
+            heartbeats_interval = (trial_heartbeats[i + 1] - trial_heartbeats[i]).total_seconds()
             assert heartbeats_interval - sleep_sec - trial_prep <= 1
```

Comment:
I realize that this value becomes negative.
What is this assertion intended to confirm?

Reply:
Thank you for your confirmation. I think negative values are not expected.
Maybe the scenario of this test should be revised.
I don't think this change resolves the error, so let me close it.